### ISSUE TITLE
how to convert picture in base64

### ISSUE THREAD

**Issue by gionnyFuego** [documents4j/documents4j#96]
Hi,
Which is the interface to implement picture as base64 string when converting doc to html ?

**Comment by raphw** [comment]
I don't think this is supported at the moment. You would need to implement your own `IConverter`.
